wordpress-mobile/WordPress-Android
# Update libs/mocks subtree with WPiOS updates

Now that WPiOS uses the same mocks as in `libs/mocks` (see https://github.com/wordpress-mobile/WordPress-iOS/pull/11788), we can pull those changes back in here as a subtree.

This is a very large diff but almost all of it has all been reviewed already. The vast majority of the changes are from https://github.com/wordpress-mobile/WordPressMocks/pull/3.

The only unreviewed changed are in this commit: https://github.com/wordpress-mobile/WordPress-Android/pull/10040/commits/0c1ba96526d997bdc289fa43743bacd21b150c37

To test:

- Run the UI tests and confirm they still pass as exptected.

Update release notes:
